src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+
+## What is a light client {#what-is-a-light-client}
+
+A light node is a node running light client software. Instead of keeping local copies of the blockchain data and independently verifying all the changes, they request the necessary data from some provider instead. The provider might be a full node or a centralized RPC server. The data is then verified by the light node, allowing it to keep up with the head of the chain. The light node only processes block headers, only occasionally downloading the actual block contents. Nodes can vary in their lightness, depending upon whether the combinations of light and full client software they run. For example, the lightest configuration would be to run a light execution client and a light consensus client. It is also likely that many nodes will choose to run light consensus clients with full execution clients, or vice-versa.
+
+## How do light clients work? {#how-do-light-clients-work}
+
+When Ethereum started using a proof-of-stake based consensus mechanism, new infrastructure was introduced specifically to support light clients. The way it works is by randomly selecting a subset of 512 validators every 1.1 days to act as a "sync-committee". The sync-committee signs the header of recent blocks. Each block header contains the the aggregated signature of the validators in the sync-committee and a "bitfield" that shows which validators signed and which did not. Each header also includes a list of validators expected to participate in signing the next block. This means a light client can quickly see that the sync-committee has signed off on the data they receive, and they can also check that the sync-committee is the genuine one by comparing the one they receive from the one they were told to expect in the previous block. In this way, the light client can keep updating it's knowledge of the latest Ethereum block without actually downloading the block itself, just the header which contains summary information.
+
+On the execution layer there is no single specification for a light execution client. The scope of a light execution client can vary from a "light mode" of a full execution client that has all the EVM and networking functionality of a full node but only verifies block headers, without downloading the associated data, or it can a more stripped down client that relies heavily upon forwarding requests to an RPC provider to interact with Ethereum.
```

Comment:
```suggestion
On the execution layer there is no single specification for a light execution client. The scope of a light execution client can vary from a "light mode" of a full execution client that has all the EVM and networking functionality of a full node but only verifies block headers, without downloading the associated data, or it can be a more stripped down client that relies heavily upon forwarding requests to an RPC provider to interact with Ethereum.
```